theano/sandbox/cuda/tests/test_basic_ops.py

```diff
@@ -380,17 +380,6 @@ def test_alloc_empty():
     assert out.shape == (2, 3)
     assert out.dtype == 'float32'
 
-    # Test that we do not merge them.
-    f = theano.function([], [cuda.basic_ops.gpu_alloc_empty(2, 3),
-                             cuda.basic_ops.gpu_alloc_empty(2, 3)])
-    out = f()
-    assert out[0].shape == (2, 3)
-    assert out[0].dtype == 'float32'
-    assert out[1].shape == (2, 3)
-    assert out[1].dtype == 'float32'
-    assert len([node for node in f.maker.fgraph.apply_nodes
-                if isinstance(node.op, cuda.basic_ops.GpuAllocEmpty)]) == 1
```

Comment:
Is the removal of this part of the test due to the change @nouiz mentionned regarding how allocs_empty are {merged/not merged}? If the alloc empty should now be merged instead of not merged, the test should be updated to reflect that.